app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
ko cần check nil đâu nha, vì find_by nil thì cũng return nil thôi
```suggestion
    @current_user ||= User.find_by(id: session[:user_id])
```